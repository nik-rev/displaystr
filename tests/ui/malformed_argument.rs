#[displaystr::display(docs)]
enum UnitVariant {
    A = "unit variant",
}

fn main() {}
