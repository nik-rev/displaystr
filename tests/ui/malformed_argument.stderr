error: unexpected token
 --> tests/ui/malformed_argument.rs:1:23
  |
1 | #[displaystr::display(docs)]
  |                       ^^^^
